src/App.tsx

```diff
@@ -1,7 +1,33 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+interface State {
+  key: string;
+}
+
+export class App extends Component<{}, State> {
+  state: Readonly<State> = {
```

Comment:
TS will infer it from initial object, you can safely go with this
```suggestion
  state = {
```

Reply:
also change `key` to `keyPressed` as per task description